data_visualization_in_open_source_software/data_visualization.md

```diff
@@ -0,0 +1,265 @@
+<!--
+
```

Comment:
can't figure out how to leave a comment on a non-code file, but the [binder_start.gif](https://github.com/arcus/education_modules/pull/2/files#diff-513bc231d2834294df3b8f80d070b08aa5df567c0dce9ba7f05c1b6625452516) can be removed from the directory as it no longer applies here.